frontend/src/features/properties/list/PropertyListView.scss

```diff
@@ -57,7 +92,7 @@ $contentheading-height: 60px;
   .ScrollContainer {
     padding: 1rem 2rem;
     max-height: calc(
-      100vh - #{$header-height} - #{$navbar-height} - #{$filter-height} - #{$footer-height}
+      100vh - #{$header-height} - #{$navbar-height} - #{$mapfilter-height} - #{$footer-height}
```

Comment:
Did you you rename the class to `map-filter`?

Reply:
No, I just used the _variables.scss file to get the values rather than the definitions in this file, it's the same value it just had a different name